source/content/shibboleth-sso.md

```diff
@@ -77,30 +71,45 @@ Commands below require a [nested docroot](/nested-docroot) structure and should
 
  ```bash{promptUser: user}
  mkdir private
- cp vendor/simplesamlphp/simplesamlphp/config-templates/config.php private/simplesaml-config.php
+ cp ./vendor/simplesamlphp/simplesamlphp/config-templates/config.php ./private/simplesaml-config.php
  ```
 
-4. Follow the directions above to [set up your config file](#configure-simplesamlphp) (`private/simplesaml-config.php`).
+4. Follow the directions in the next section to [set up your config file](#configure-simplesamlphp) (`private/simplesaml-config.php`).
 
 5. Add a symlink from SimpleSAMLphp's default config file over to your customized config, stored outside the vendor directory:
 
  ```bash{promptUser: user}
- ln -s ../../../../private/simplesaml-config.php ./vendor/simplesamlphp/simplesamlphp/config/config.php
+ ln -s ../private/simplesaml-config.php ./vendor/simplesamlphp/simplesamlphp/config/config.php
  ```
 
 6. Add this symlink as a post-update script to `composer.json`. This allows the symlink to be recreated if we update or re-install SimpleSAMLphp using Composer:
 
  ```json:title=composer.json
    "scripts": {
        "post-install-cmd": [
-           "ln -s ../../../../private/simplesaml-config.php ./vendor/simplesamlphp/simplesamlphp/config/config.php"
+           "ln -sf ../private/simplesaml-config.php ./vendor/simplesamlphp/simplesamlphp/config/config.php"
```

Comment:
@kyletaylored should this `../private` be `./private`, since we're still operating from the site root?

Reply:
I thought so too, but it's because the actual path being symlinked is being called through `/web/index.php`, so to load the config relative to `/web`, it will be `../private/`.